examples/mundialis_example.py

```diff
@@ -1,21 +1,29 @@
-import openeo
-import logging
+#!/usr/bin/env python3
+
 import json
+import logging
+import openeo
+from openeo.rest.imagecollectionclient import ImageCollectionClient
 
 logging.basicConfig(level=logging.INFO)
 
 
-MUNDIALIS_DRIVER_URL = "https://openeo.mundialis.de/api/v0.4/"
+backend_url = 'https://openeo.mundialis.de'
+auth_id = "openeo"
+auth_pwd = "FIXME"
 
-OUTPUT_FILE = "/tmp/openeo_mundialis_output.png"
+collection_id = 'utm32n.openeo_bolzano.strds.openeo_bolzano_S2'
 
+outfile = "/tmp/openeo_mundialis_output.png"
 
-con = openeo.connect(MUNDIALIS_DRIVER_URL)
+# connect with mundialis openeo/actinia backend
+con = openeo.connect(backend_url).authenticate_basic(auth_id, password = auth_pwd)
 
-#Test Connection
+# Test Connection
 print(con.list_processes())
 print(con.list_collections())
-print(con.describe_collection("nc_spm_08.modis_lst.strds.MOD11B3.A2016306.h11v05.single_LST_Day_6km"))
+print(con.describe_collection("utm32n.openeo_bolzano.strds.openeo_bolzano_S2"))
+print(con.describe_collection("nc_spm_08.modis_lst.strds.MOD11B3"))
```

Comment:
nc_spm_08.modis_lst.strds.MOD11B3 does not exist
use nc_spm_08.modis_lst.strds.LST_Day_monthly